django/db/models/sql/query.py

```diff
@@ -1299,13 +1299,32 @@ def build_filter(self, filter_expr, branch_negated=False, current_negated=False,
             self.check_filterable(value)
 
         clause = self.where_class()
+        alias = self.get_initial_alias()
         if reffed_expression:
             condition = self.build_lookup(lookups, reffed_expression, value)
             clause.add(condition, AND)
-            return clause, []
+            targets = reffed_expression.get_source_expressions()
+            if targets and isinstance(targets[0], Col):
+                target = targets[0].target
+                lookup_type = condition.lookup_name
+                clause.add(condition, AND)
+                if current_negated and lookup_type != 'isnull' and condition.rhs is not None \
+                        and self.is_nullable(target):
```

Comment:
Avoid line continuation characters.
```suggestion
                if (
                        current_negated and
                        lookup_type != 'isnull' and
                        condition.rhs is not None and
                        self.is_nullable(target)
                ):

Reply:
Thanks! I updated.

Reply:
This appears to have resurfaced.

Reply:
Yes. I changed this by felixxm [suggestion](https://github.com/django/django/pull/14065#discussion_r635015330).

Reply:
Oh I just meant code style. The suggestion didn't have a backslash, and we avoid them in Django.

Reply:
ah okay! Thanks!